t/034-validate-reference-count.t.cpp

```diff
@@ -0,0 +1,188 @@
+#include "clangmetatool-testconfig.h"
```

Comment:
This file is not being built, and also seems to be largely a copy of `031-validate-include-graph.t.cpp`. Was it added accidentally?

Reply:
Whoops, yeah, shouldn't have been committed.